src/store/user.js

```diff
@@ -46,7 +45,7 @@ const mutations = {
 const actions = {
   sendToken({ commit, dispatch }, userdata) {
     return new Promise((resolve, reject) => {
-      login(userdata)
+      UserAPI.login(userdata.idToken)
```

Comment:
Why we pass `userdata.idToken`? I'm not sure if  we have modified BE API data parameter

Reply:
I see. [Here](https://github.com/NCTU-SYNC/sync-server/blob/master/src/controllers/auth_controller.js#L16) requires whole `userdata` object.

Problem fix. ([commit](https://github.com/NCTU-SYNC/sync-frontend/commit/0d15d8c66cfebb3db10aaf440389b317a49fc64e))